ReactViewControl/ReactView.cs

```diff
@@ -77,6 +77,11 @@ public abstract partial class ReactView : IDisposable {
         /// </summary>
         protected virtual ReactViewFactory Factory => new ReactViewFactory();
 
+        protected void RefreshDefaultStyleSheet() {
+            CachedViews.Clear();
```

Comment:
this line has to be executed in the ui thread, otherwise it might cause concurrency problems accessing the cachedviews

Reply:
you should only remove the entry of the current factory type..,

Reply:
I was not aware of those Factory types. Thanks